options/ansi/generic/string-stubs.cpp

```diff
@@ -414,7 +414,144 @@ int strverscmp(const char *l0, const char *r0) {
 	return l[i] - r[i];
 }
 
-void *memmem(const void *, size_t, const void *, size_t) {
-	__ensure(!"Not implemented");
-	__builtin_unreachable();
+// Taken from musl.
+static char *twobyte_memmem(const unsigned char *h, size_t k, const unsigned char *n) {
```

Comment:
These can rather be in an anonymous namespace. It's the appropriate style in mlibc.